cli/cmd/plugin/standalone-cluster/cluster/kind.go

```diff
@@ -88,6 +88,16 @@ func (kcm KindClusterManager) Delete(c *config.LocalClusterConfig) error {
 	return provider.Delete(c.ClusterName, "")
 }
 
+// PullImage will fetch a container image to the cluster host.
+func (kcm KindClusterManager) PullImage(imageName string) error {
+	cmd := exec.Command("docker", "pull", imageName)
```

Comment:
Yeah, that was one of my concerns with this. I'm glad you brought it up on here.

I _think_ we are OK. Kind will interact with either docker or podman, so without this (and the antrea patching), we should work just fine with either container engine.

I figured this use would be OK since we do have the antrea code also calling out, but ideally I think we wouldn't want any exec calls here. But one saving thing in all this is I think many deployments `alias docker=podman`, and for most operations they are compatible. So if they are not already doing that, it would be a fairly easy workaround to get this step working.

I'm open to just closing this PR for now too. We could wait and see if this is a big enough issue that we need to take extra steps like this. Ideally we just make the calls out to kind and leave it at that.

Reply:
Not necessarily. Keep in mind this implementation is specific to the `KindClusterManager`.

The reason I am OK with this, is that kind itself calls out to docker this way, thus the dependency for our `--provider=kind` does not change.